src/app/lib/config/services/azeri.ts

```diff
@@ -302,6 +302,14 @@ export const service: DefaultServiceConfig = {
           href: 'https://www.bbc.co.uk/azeri/send/u50853225',
           text: 'BBC ilə Əlaqə',
         },
+        {
+          href: 'https://www.bbc.com/ws/languages',
+          text: 'Other Languages',
+        },
+        {
+          href: 'https://www.bbc.com/ws/languages',
+          text: 'Other Languages',
+        },
```

Comment:
Double here

Reply:
So good I linked it twice. Will remove these overexcited copy pastes.